src/class/IContentTypes/IContentTypes.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef I_CONTENT_TYPES_HPP
+#define I_CONTENT_TYPES_HPP
+
+#include <iostream>
+#include <list>
+#include <map>
+#include <string>
+
+#ifdef TEST_ON
+#define private public
+#define protected public
+#endif
+
+class IContentTypes
+{
+  public:
+	static std::map<std::string, std::string> _content_types;
+	static std::map<std::string, std::string> create_map();
```

Comment:
i think this func can be just a normal "static" like we had in C -> only exist in the .cpp file, no declaration is needed

Reply:
great idea!